requirements.txt

```diff
@@ -41,14 +42,15 @@ pytest-mock==1.10.0
 python-engineio==2.1.0
 python-ldap==3.0.0
 python-socketio==1.9.0
-python3-saml==1.4.1
+python3-saml==1.3.0
```

Comment:
Why is this version lower? 1.4 fixes a vulnerability so we should always be using the most up to date version.

Reply:
Updated back to 1.4.1 and got rid of SLO because RIT-Shib doesn't implement it.